src/analyzers/index.js

```diff
@@ -0,0 +1,18 @@
+class Analyzer {
+  constructor(project) {
+    this._project = project;
+  }
+
+  get containsExternalScript() {
+    const docElement = this._project.get('sources').get('html');
+    const doc = new DOMParser().parseFromString(docElement, 'text/html');
```

Comment:
Initialize both in the constructor - this'll get us some savings should we eventually ask any more questions of the html.